source/speech/speech.py

```diff
@@ -3059,3 +3063,38 @@ def clearTypedWordBuffer() -> None:
 	complete the word (such as a focus change or choosing to move the caret).
 	"""
 	_curWordChars.clear()
+
+
+def _sapi4DeprecationWarning(synth: SynthDriver, audioOutputDevice: str, isFallback: bool):
```

Comment:
Wouldn't this fit better in the sapi4 module? Note that the module will always be loaded pre-init, so the extension point registration will work.